### ISSUE TITLE
INVALID_TYPE: sObject type 'ReleaseUpdate' is not supported.

### ISSUE THREAD

**Issue by ghost** [jsforce/jsforce#1268]
I am trying to run a tooling API query that I have run before and can run in Salesforce Inspector or Postman. However, I am now getting an error from jsforce when running it.

Tooling API Query:
```select DurableId, Title, ReleaseLabel, DueDate, Category, Description, HasNewSteps, IsReleased from ReleaseUpdate ORDER BY DueDate```

>INVALID_TYPE: sObject type 'ReleaseUpdate' is not supported.
>    at HttpApi.getError (/Users/Tim.Paulaskas/git-repos/Monitor/node_modules/jsforce/lib/http-api.js:250:13)
>    at /Users/Tim.Paulaskas/git-repos/Monitor/node_modules/jsforce/lib/http-api.js:95:22
>    at tryCallOne (/Users/Tim.Paulaskas/git-repos/Monitor/node_modules/promise/lib/core.js:37:12)
>    at /Users/Tim.Paulaskas/git-repos/Monitor/node_modules/promise/lib/core.js:123:15
>    at flush (/Users/Tim.Paulaskas/git-repos/Monitor/node_modules/asap/raw.js:50:29)
>    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
>  errorCode: 'INVALID_TYPE'
>}`

This is the result I get from Postman:
{{_endpoint}}/services/data/v{{version}}/tooling/query/?q=select DurableId, Title, ReleaseLabel, DueDate, Category, Description, HasNewSteps, IsReleased from ReleaseUpdate ORDER BY DueDate

```
{
    "size": 53,
    "totalSize": 53,
    "done": true,
    "queryLocator": null,
    "entityTypeName": "ReleaseUpdate",
    "records": [
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AuraSecStaticResCRUC"
            },
            "DurableId": "AuraSecStaticResCRUC",
            "Title": "Enable Secure Static Resources for Lightning Components",
            "ReleaseLabel": null,
            "DueDate": null,
            "Category": "Security",
            "Description": "This release update has been postponed indefinitely while we change the implementation to reduce customer impact. The release update won’t be enforced in its present form. Don’t enable it.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/GuestUserEmailIdUpdate"
            },
            "DurableId": "GuestUserEmailIdUpdate",
            "Title": "Update Guest User’s Email Address",
            "ReleaseLabel": null,
            "DueDate": null,
            "Category": "Security",
            "Description": "When you create an Experience Cloud or Salesforce site, Salesforce automatically creates a guest site user. In any site created before the Summer ’21 release, the email address for the site guest user record was that of the Salesforce user who created the site. After the Summer ’21 release, the system generates a dummy email address to use in the guest site user record.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/SDCAdoptionReleaseUpdate"
            },
            "DurableId": "SDCAdoptionReleaseUpdate",
            "Title": "Enable Content Delivery Network for Digital Experiences",
            "ReleaseLabel": null,
            "DueDate": null,
            "Category": "Usability",
            "Description": "Use CDN for Digital Experiences to ensure high performance of your site during peak traffic events. Allow Salesforce to manage your certificates for you.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ApexNoArgConstructorAccessCriticalUpdate"
            },
            "DurableId": "ApexNoArgConstructorAccessCriticalUpdate",
            "Title": "Restrict Reflective Access to Non-Global Constructors in Packages",
            "ReleaseLabel": "Summer '20 Patch 18.0",
            "DueDate": "2020-08-19",
            "Category": "Other",
            "Description": "When this update is enabled, regardless of API version, you can initiate only Apex classes that have a no-arguments constructor that is visible to the code running Type.newInstance.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/MigrateLegacyTransactionSecurityCRUC"
            },
            "DurableId": "MigrateLegacyTransactionSecurityCRUC",
            "Title": "Migrate Legacy Policies to the Enhanced Transaction Security Framework",
            "ReleaseLabel": "Winter '21",
            "DueDate": "2020-09-10",
            "Category": "EndOfLife",
            "Description": "We're retiring the legacy transaction security framework in the Winter '20 release.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AuraApexCtrlAuthUserAccessCheckCRUC"
            },
            "DurableId": "AuraApexCtrlAuthUserAccessCheckCRUC",
            "Title": "Restrict Access to @AuraEnabled Apex Methods for Authenticated Users Based on User Profile",
            "ReleaseLabel": "Winter '21",
            "DueDate": "2020-09-10",
            "Category": "Security",
            "Description": "This update gives you more control over which authenticated users can access Apex classes containing @AuraEnabled methods. When this update is activated, an authenticated user can access an @AuraEnabled Apex method only when the user’s profile allows access to the Apex class.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AuraApexCtrlGuestUserAccessCheckCRUC"
            },
            "DurableId": "AuraApexCtrlGuestUserAccessCheckCRUC",
            "Title": "Restrict Access to @AuraEnabled Apex Methods for Guest and Portal Users Based on User Profile",
            "ReleaseLabel": "Winter '21",
            "DueDate": "2020-09-10",
            "Category": "Security",
            "Description": "This update gives you more control over which guest, portal, or community users can access Apex classes containing @AuraEnabled methods. Add guest user profile access to any @AuraEnabled Apex class used by a community or portal.<br/>When this update is activated, a guest, portal, or community user can access an @AuraEnabled Apex method only when the user’s profile allows access to the Apex class.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AllowStdExtProfileLoginCRUC"
            },
            "DurableId": "AllowStdExtProfileLoginCRUC",
            "Title": "Allow Users to Use Standard External Profiles to Self-Register and Log In To Experience Cloud Sites",
            "ReleaseLabel": "Spring '21 Patch 18.0",
            "DueDate": "2021-01-06",
            "Category": "Other",
            "Description": "This update allows users to self-register with standard external profiles, and allows users assigned to those profiles to log in to Experience Cloud sites.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/InvocableActionPartialSaveCRUC"
            },
            "DurableId": "InvocableActionPartialSaveCRUC",
            "Title": "Enable Partial Save for Invocable Actions",
            "ReleaseLabel": "Spring '21",
            "DueDate": "2021-01-10",
            "Category": "Usability",
            "Description": "This update prevents a failed invocable action from causing the entire transaction to fail and prevents the execution of other invocable actions in the failed transaction. With this update, Salesforce rolls back only the records whose updates get blocked by failed invocable actions.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowPluginModifiersCRUC"
            },
            "DurableId": "FlowPluginModifiersCRUC",
            "Title": "Make Flows Respect Access Modifiers for Legacy Apex Actions",
            "ReleaseLabel": "Spring '21",
            "DueDate": "2021-01-10",
            "Category": "Security",
            "Description": "With this update enabled, flows fail if they contain a public legacy Apex action.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/CustomNotifActionAccessCheckReleaseUpdate"
            },
            "DurableId": "CustomNotifActionAccessCheckReleaseUpdate",
            "Title": "Require User Permission for the Send Custom Notification Action",
            "ReleaseLabel": "Spring '21",
            "DueDate": "2021-01-10",
            "Category": "Security",
            "Description": "This update gives you more control over which of your users can trigger the Send Custom Notification action in most flows, REST API, and Apex. When this update is activated, a user must have the Send Custom Notifications user permission to trigger the Send Custom Notification action in flows that run in user context, REST API calls, or Apex callouts.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AuraSecureEvalCRUC"
            },
            "DurableId": "AuraSecureEvalCRUC",
            "Title": "Prevent Creation of Function Expressions When Using $A.createComponent() or $A.createComponents() in Aura Components.",
            "ReleaseLabel": "Spring '21",
            "DueDate": "2021-01-10",
            "Category": "Security",
            "Description": "To improve security and stability, this update prevents attribute values passed to $A.createComponent() or $A.createComponents() from being interpreted as Aura function expressions.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/EnforceDataAccessInFlowFormulas"
            },
            "DurableId": "EnforceDataAccessInFlowFormulas",
            "Title": "Enforce Data Access in Flow Merge Fields",
            "ReleaseLabel": "Summer '21",
            "DueDate": "2021-05-10",
            "Category": "Security",
            "Description": "This update enforces the running user’s data access when Flow Builder uses a merge field to access a field on a related record.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/MngdCtrlActionAccessCRUC"
            },
            "DurableId": "MngdCtrlActionAccessCRUC",
            "Title": "Disable Access to Non-global Controller Methods in Managed Packages",
            "ReleaseLabel": "Summer '21",
            "DueDate": "2021-05-10",
            "Category": "Security",
            "Description": "This update corrects access controls on Apex controller methods in managed packages. When this update is enabled, only methods marked with the global access modifier are accessible by Aura components from outside the package namespace.<br/>These access controls prevent you from using unsupported API methods that the package author didn’t intend for global access.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ExternalServicesRetireV1CRUC"
            },
            "DurableId": "ExternalServicesRetireV1CRUC",
            "Title": "Migrate Legacy External Services to the Current External Services Version",
            "ReleaseLabel": "Summer '21",
            "DueDate": "2021-05-10",
            "Category": "EndOfLife",
            "Description": "Legacy External Services begins its phased retirement in Spring '21. To prepare for this retirement and take advantage of the new features, migrate your Legacy External Services to the current External Services version as soon as possible.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ApexPropertyGetterCRUC"
            },
            "DurableId": "ApexPropertyGetterCRUC",
            "Title": "Enforce Access Modifiers on Apex Properties in Lightning Component Markup",
            "ReleaseLabel": "Summer '21",
            "DueDate": "2021-05-10",
            "Category": "Security",
            "Description": "This update makes Lightning components consistent with the use of Apex properties in other contexts. For example, a markup expression can no longer access an Apex property with a private Apex getter.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/RemoveTM1GroupMemberCRUC"
            },
            "DurableId": "RemoveTM1GroupMemberCRUC",
            "Title": "Delete Your Original Territory Management Sharing Records",
            "ReleaseLabel": "Winter '22",
            "DueDate": "2021-09-10",
            "Category": "EndOfLife",
            "Description": "Now that the original territory management feature was retired, this update deletes your original territory management sharing records to prevent issues in Enterprise Territory Management.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AuraComponentAccessFix"
            },
            "DurableId": "AuraComponentAccessFix",
            "Title": "Enable Aura Component Access Fix",
            "ReleaseLabel": "Winter '22",
            "DueDate": "2021-09-10",
            "Category": "Security",
            "Description": "This update fixes a bug in Aura‘s Component access controls.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/RecognitionBadgeGuestUserAccessReleaseUpdate"
            },
            "DurableId": "RecognitionBadgeGuestUserAccessReleaseUpdate",
            "Title": "Removal of Guest User Access to Recognition Badges in Experience Builder Sites",
            "ReleaseLabel": "Winter '22",
            "DueDate": "2021-09-10",
            "Category": "Security",
            "Description": "To improve security, we're disabling guest user access to recognition badges in Experience Builder sites by default.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/VisualforceJSRemotingEnforcement"
            },
            "DurableId": "VisualforceJSRemotingEnforcement",
            "Title": "Enable Visualforce JavaScript Remoting Enforcement",
            "ReleaseLabel": "Winter '22",
            "DueDate": "2021-09-10",
            "Category": "Security",
            "Description": "This update will throw an error if the Visualforce JavaScript <code>$RemoteAction.ApexController.method</code> function is invoked improperly.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/LightningOAuthScopeSecurityFix"
            },
            "DurableId": "LightningOAuthScopeSecurityFix",
            "Title": "Enforce OAuth Scope for Lightning Apps",
            "ReleaseLabel": "Winter '22",
            "DueDate": "2021-09-10",
            "Category": "Security",
            "Description": "To improve security, this update enforces the scope of an OAuth token used to authenticate a Lightning app. The scope of an OAuth token is defined in a connected app.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/PartnerBTEnforceMFAReleaseUpdate"
            },
            "DurableId": "PartnerBTEnforceMFAReleaseUpdate",
            "Title": "Require Multi-Factor Authentication For Logins to Subscriber Orgs",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2021-12-05",
            "Category": "Security",
            "Description": "Secure access to subscriber orgs by requiring multi-factor authentication (MFA) when logging into the License Management Org (LMO).",
            "HasNewSteps": true,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/VisualforceELApexModifierEnforcement"
            },
            "DurableId": "VisualforceELApexModifierEnforcement",
            "Title": "Enable Visualforce Expression Language Apex Method Access Modifier Enforcement",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-01-10",
            "Category": "Security",
            "Description": "This update throws an error if the Visualforce Expression Language can't access the Apex method due to access modifier visibility.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ProductOWDReleaseUpdate"
            },
            "DurableId": "ProductOWDReleaseUpdate",
            "Title": "Set Organization-Wide Sharing Defaults for Products",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-01-10",
            "Category": "Security",
            "Description": "To better protect your data, this update does the following:\n            \n            <ul>\n                <li>In Winter ’22, adds organization-wide sharing settings for product records</li>\n                <li>As of Spring ’22, in new orgs only, sets the organization-wide sharing default for products to Private for external users</li>\n            </ul>",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/DisableForcedLoginReleaseUpdate"
            },
            "DurableId": "DisableForcedLoginReleaseUpdate",
            "Title": "Disable Users from Logging Into an Org or Experience Cloud Site with Login Credentials as URL Query String Parameters",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-01-10",
            "Category": "Security",
            "Description": "Salesforce is removing the ability for users to log into orgs or Experience Cloud sites with usernames and passwords as query string parameters to the login URL.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/AuraCorsAllowlistSecurityFix"
            },
            "DurableId": "AuraCorsAllowlistSecurityFix",
            "Title": "Enforce CORS Allowlist for Lightning Apps",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-01-10",
            "Category": "Security",
            "Description": "<p>The Cross-Origin Resource Sharing (CORS) allowlist in Setup isn’t currently enforced for Lightning apps. This update enforces the allowlist. Cross-origin requests to Lightning apps are blocked unless the request comes from a URL listed in your org’s CORS allowlist.</p>\n        <br/>\n        <p>CORS enables web browsers to request resources from origins other than their own. For example, using CORS, JavaScript code at https://www.example.com could request a resource from https://www.salesforce.com. To access Lightning Out from JavaScript code in a web browser, add the origin requesting the code to your CORS allowlist in Setup.</p>",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ProductOWDGuestUserReleaseUpdate"
            },
            "DurableId": "ProductOWDGuestUserReleaseUpdate",
            "Title": "Retain Guest Users’ Access to Products",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-01-10",
            "Category": "Security",
            "Description": "To better protect your Salesforce org, this update changes how unauthenticated guest access to product records (the Product2 object) is controlled. The product organization-wide sharing default for external users is no longer applied to guest users. The organization-wide default is set to Private for guest users and can’t be changed. To retain guest users’ access to products, create guest user sharing rules.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowNullInFormulaCRUC"
            },
            "DurableId": "FlowNullInFormulaCRUC",
            "Title": "Check for Null Record Variables or Null Values of Lookup Relationship Fields in Process and Flow Formulas",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-01-10",
            "Category": "Usability",
            "Description": "This update enables process and flow formulas to reference a record’s field when the field is accessed via a null lookup relationship field or null record variable.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowApexCpuTimingReleaseUpdate"
            },
            "DurableId": "FlowApexCpuTimingReleaseUpdate",
            "Title": "Accurately Measure the CPU Time Consumption of Flows and Processes",
            "ReleaseLabel": "Summer '22",
            "DueDate": "2022-05-10",
            "Category": "Usability",
            "Description": "This update lets Salesforce accurately measure, log, and limit the CPU time consumed by flows and processes.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/RqrEmlChgVerfAdmDrvnECRPReleaseUpdate"
            },
            "DurableId": "RqrEmlChgVerfAdmDrvnECRPReleaseUpdate",
            "Title": "Securely Update Email Addresses and Reset Passwords",
            "ReleaseLabel": "Summer '22",
            "DueDate": "2022-05-10",
            "Category": "Security",
            "Description": "To ensure the security of your org, users must now reset their password before your changes to their email address and password become active. When a user resets the password using the provided link, the new email address is activated. Previously, the user’s email address became active as soon as you saved the change.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/SuppressMfaForOidcSsoReleaseUpdate"
            },
            "DurableId": "SuppressMfaForOidcSsoReleaseUpdate",
            "Title": "Bypass MFA Challenges for Single Sign-On Auth Provider Logins",
            "ReleaseLabel": "Summer '22",
            "DueDate": "2022-05-10",
            "Category": "Security",
            "Description": "This release update will enforce the documented behavior of bypassing MFA challenges for users who are assigned the user permission Multi-factor Authentication for User Interface Logins.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ApiVersions7to20Retirement"
            },
            "DurableId": "ApiVersions7to20Retirement",
            "Title": "Salesforce Platform API Versions 7.0 through 20.0 Retirement",
            "ReleaseLabel": "Summer '22",
            "DueDate": "2022-05-10",
            "Category": "EndOfLife",
            "Description": "Versions 7.0 through 20.0 of the Salesforce Platform SOAP, REST, and Bulk APIs will be deprecated and no longer supported in the Summer '21 release. Applications should be modified to consume the latest available API version.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/LLoginFlowWarningReleaseUpdate"
            },
            "DurableId": "LLoginFlowWarningReleaseUpdate",
            "Title": "Summer ‘22 - You can now disable the rendering of new Login Flows in Lightning Runtime.",
            "ReleaseLabel": "Spring '22",
            "DueDate": "2022-05-20",
            "Category": "Security",
            "Description": "As of the Summer ‘22 release, customers can improve security by disabling the rendering of new Login Flows in Lightning Runtime.",
            "HasNewSteps": false,
            "IsReleased": true
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/VisualforceELApexModifierGetSetEnforcement"
            },
            "DurableId": "VisualforceELApexModifierGetSetEnforcement",
            "Title": "Validate Getter and Setter Access Modifier Enforcement During Visualforce Expression Language Parsing",
            "ReleaseLabel": "Winter '23",
            "DueDate": "2022-09-10",
            "Category": "Security",
            "Description": "This update throws an error if the Visualforce Expression Language can’t validate the access modifier for the Apex get or set method.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/VisualforceJSRemotingEnforcementWithPerm"
            },
            "DurableId": "VisualforceJSRemotingEnforcementWithPerm",
            "Title": "Enable Visualforce JavaScript Remoting Enforcement",
            "ReleaseLabel": "Winter '23",
            "DueDate": "2022-09-10",
            "Category": "Security",
            "Description": "This update will throw an error if the Visualforce JavaScript <code>$RemoteAction.ApexController.method</code> function is invoked improperly.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/EnableEnhancedDomainsReleaseUpdate"
            },
            "DurableId": "EnableEnhancedDomainsReleaseUpdate",
            "Title": "Enable Enhanced Domains",
            "ReleaseLabel": "Winter '23",
            "DueDate": "2022-09-10",
            "Category": "Security",
            "Description": "To comply with the latest browser and security standards, enable enhanced domains on your Salesforce org’s My Domain. This update affects application and login URL formats across your org and is enforced in phases beginning in Winter ’23. This update is enforced in sandboxes and non-production orgs in Winter ’23 unless you enable the postponement setting on the My Domain Setup page. This update is enforced in production orgs and all remaining orgs in Spring ’23.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/EnablePSAExpirationUI"
            },
            "DurableId": "EnablePSAExpirationUI",
            "Title": "Enable Permission Set Expiration and Enhanced UI",
            "ReleaseLabel": "Winter '23",
            "DueDate": "2022-09-10",
            "Category": "Security",
            "Description": "With this update enabled, permission set and permission set group assignments use a new Lightning interface and an improved workflow. Also, you have the option to assign an expiration date to each permission set and permission set groups",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/GuestUserPSLUpdate"
            },
            "DurableId": "GuestUserPSLUpdate",
            "Title": "Remove Guest User Assignments from Permission Sets Associated with Permission Set Licenses with Restricted Object Permissions",
            "ReleaseLabel": "Winter '23",
            "DueDate": "2022-09-10",
            "Category": "Security",
            "Description": "<p>To improve the security of your data, Salesforce is removing guest user assignments from permission sets and permission set groups associated with permission set licenses that contain View All, Modify All, edit, and delete object permissions. After the permission set or permission set group assignment is removed, guest user assignments of permission set licenses with these restricted permissions have no effect. The only object permissions allowed for guest users are read and create. Going forward, you can no longer assign guest users permission sets or permission set groups that are associated with these permission set licenses and contain the restricted permissions.</p>\n        </br>\n        <p>Note: This update was originally called \"Remove Permission Set Licenses with View All, Modify All, Edit, and Delete Object Permissions from Guest Users.\"</p>",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/EnhancedPersonalInformationManagementReleaseUpdate"
            },
            "DurableId": "EnhancedPersonalInformationManagementReleaseUpdate",
            "Title": "Enable Stronger Protection for Your Users’ Personal Information",
            "ReleaseLabel": "Winter '23",
            "DueDate": "2022-09-10",
            "Category": "Security",
            "Description": "To prevent external users, such as portal or guest users, from viewing other users’ personal information, enable Enhanced Personal Information Management. This feature replaces the Hide Personal Information setting and allows you to specify which fields are considered PII.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ExperienceDisableRunFlowsReleaseUpdate"
            },
            "DurableId": "ExperienceDisableRunFlowsReleaseUpdate",
            "Title": "Require Granular Flow Permissions for Experience Cloud Guest Users",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Security",
            "Description": "This update removes the Run Flows permission from the guest user profiles associated with Experience Cloud sites. The change improves site security by requiring explicit guest user permissions to run flows.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/NewOrderSaveLogicCRUC"
            },
            "DurableId": "NewOrderSaveLogicCRUC",
            "Title": "Enable New Order Save Behavior",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Other",
            "Description": "After the New Order Save Behavior is enabled, whenever an order product update causes a change to the parent order, Salesforce runs custom application logic. Previously, in this scenario, Salesforce didn’t correctly evaluate custom application logic on the parent record. Users will have the option to enable the new behavior in Winter &#39;21. Salesforce will automatically enforce the updated behavior in Spring &#39;23.\n                <br/><br/>Warning: Packages with orders created before Winter &#39;21 don't support the New Order Save Behavior release update by default. If you use a package involving orders, wait for confirmation from your package provider before enabling the release update.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/IsTimeResumedInSameRunContext"
            },
            "DurableId": "IsTimeResumedInSameRunContext",
            "Title": "Make Paused Flow Interviews Resume in the Same Context with the Same User Access",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Security",
            "Description": "With this update enabled, paused autolaunched flows always resume in the same context and with the same user access they had before they were paused.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/UseOpenSAMLV4ForOutboundReleaseUpdate"
            },
            "DurableId": "UseOpenSAMLV4ForOutboundReleaseUpdate",
            "Title": "Upgrade SAML Single Sign-On Framework",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Security",
            "Description": "Salesforce is upgrading its SAML framework as part of regular ongoing maintenance. This update can impact integrations with third-party systems. For example, this update can impact integrations with SAML identity providers or SAML-enabled applications. This update applies to every SAML-based integration, whether you’re using Identity for Employees or Salesforce Customer Identity, including Experience Cloud.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowViaRestUsesUserCtxtCRUC"
            },
            "DurableId": "FlowViaRestUsesUserCtxtCRUC",
            "Title": "Run Flows in User Context via REST API.",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Security",
            "Description": "With this update enabled, a flow that runs via REST API uses the running user’s profile and permission sets to determine the object permissions and field-level access of the flow.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/LocaleFormatCRUC"
            },
            "DurableId": "LocaleFormatCRUC",
            "Title": "Enable ICU Locale Formats",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Usability",
            "Description": "Changes the way that dates, times, numbers, and currencies appear, using locale formats from the International Components for Unicode (ICU) library.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/OWAForCaseEmailsAllowedReleaseUpdate"
            },
            "DurableId": "OWAForCaseEmailsAllowedReleaseUpdate",
            "Title": "Use a Default No-Reply Address as System Address for Case Email Notifications",
            "ReleaseLabel": "Spring '23",
            "DueDate": "2023-01-10",
            "Category": "Security",
            "Description": "<p>From Setup, configure a default No-Reply address from the Organization-Wide Addresses page. Add an email address in Special Purpose Organization-Wide Email Addresses, then follow the steps outlined in the verification email sent to the new default No-Reply address.</p>\n                <br/>\n                <p>After you verify your default No-Reply address, set up and start the Test Run in the Release Update.</p>",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/EmailThreadingCRUC"
            },
            "DurableId": "EmailThreadingCRUC",
            "Title": "Disable Ref ID and Transition to New Email Threading Behavior",
            "ReleaseLabel": "Summer '23",
            "DueDate": "2023-05-10",
            "Category": "Security",
            "Description": "Disable Ref ID and Use New Email-to-Case Threading.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ApiVersions21to30Retirement"
            },
            "DurableId": "ApiVersions21to30Retirement",
            "Title": "Salesforce Platform API Versions 21.0 through 30.0 Retirement",
            "ReleaseLabel": "Summer '23",
            "DueDate": "2023-05-10",
            "Category": "EndOfLife",
            "Description": "Versions 21.0 through 30.0 of the Salesforce Platform SOAP, REST, and Bulk APIs will be deprecated and no longer supported in the Summer '22 release. Applications should be modified to consume the latest available API version.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowCommunityApexContextRetiredCRUC"
            },
            "DurableId": "FlowCommunityApexContextRetiredCRUC",
            "Title": "Disable Rules for Enforcing Explicit Access to Apex Classes",
            "ReleaseLabel": "Summer '23",
            "DueDate": "2023-05-10",
            "Category": "Security",
            "Description": "With this update, the running user of a flow or process no longer requires explicit access to Apex classes that are invoked by the flow or process.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowNullPreviousValueFix"
            },
            "DurableId": "FlowNullPreviousValueFix",
            "Title": "Evaluate Criteria Based on Original Record Values in Process Builder",
            "ReleaseLabel": "Summer '23",
            "DueDate": "2023-05-10",
            "Category": "Usability",
            "Description": "This update fixes a bug with the evaluation criteria in processes that have multiple criteria and a record update.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/ScopedModalsCRUC"
            },
            "DurableId": "ScopedModalsCRUC",
            "Title": "Keep Working with Tab-Focused Dialogs",
            "ReleaseLabel": "Summer '24",
            "DueDate": "2024-05-10",
            "Category": "Usability",
            "Description": "In Lightning console apps, dialogs no longer stop you from interacting with the rest of the UI. This release update limits the focus of dialogs triggered by a workspace tab or subtab to only the tab that triggered it.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/DoubleNavApi"
            },
            "DurableId": "DoubleNavApi",
            "Title": "Prevent consecutive API navigation calls",
            "ReleaseLabel": "Summer '24",
            "DueDate": "2024-05-10",
            "Category": "Usability",
            "Description": "In Visualforce pages and Lightning components, API navigation calls can be fired consecutively. This release update prevents this API navigation call pattern, firing only the first navigation call.",
            "HasNewSteps": false,
            "IsReleased": false
        },
        {
            "attributes": {
                "type": "ReleaseUpdate",
                "url": "/services/data/v54.0/tooling/sobjects/ReleaseUpdate/FlowCommunityApexContextCRUC"
            },
            "DurableId": "FlowCommunityApexContextCRUC",
            "Title": "Require User Access to Apex Classes Invoked by Flow",
            "ReleaseLabel": "Winter '25",
            "DueDate": "2024-09-10",
            "Category": "Other",
            "Description": "This update requires a user running a flow to have access to all Apex classes invoked by that flow. If the user does not have an Apex class assignment via their profile or permission set, the flow fails when it invokes the Apex class",
            "HasNewSteps": false,
            "IsReleased": false
        }
    ]
}
```

**Comment by cristiand391** [comment]
when querying tooling objects you need to run the query via `conn.tooling.query(<query>)`, not `conn.query`.

https://jsforce.github.io/document/#query-tooling-objects
